trino/client.py

```diff
@@ -925,3 +948,103 @@ def _parse_retry_after_header(retry_after):
         retry_date = parsedate_to_datetime(retry_after)
         now = datetime.utcnow()
         return (retry_date - now).total_seconds()
+
+
+# Trino Spooled protocol transfer objects
+SpooledSegmentMetadata = TypedDict('SpooledSegmentMetadata', {'uncompressedSize': str})
+SpooledSegment = TypedDict(
+    'SpooledSegment',
+    {
+        'type': str,
+        'uri': str,
+        'ackUri': NotRequired[str],
```

Comment:
I believe that in the final version of the protocol, this has become required (I know that it could be missing in some earlier versions of Trino).

Reply:
yes, it is required